client/src/game/states.ts

```diff
@@ -10,5 +10,6 @@ export const states = {
     gameSocket: undefined as (Socket | undefined),
     lobby: undefined as (types.LobbyId | undefined),
     id: undefined as (types.ClientId | undefined),
-    name: undefined as (string | undefined)
+    name: undefined as (string | undefined),
+    lastMessage: ref<string>()
```

Comment:
Which component will re-render when you set this state?

Reply:
The `App.vue` component. Later I'll probably create a separate message / notification component that'll update by itself.